trading_report/src/main.ts

```diff
@@ -12,7 +12,7 @@ async function main() {
   console.log('Analysis running', new Date().toUTCString(), SYMBOL);
 
   const connection = new Connection(
-    'https://floral-skilled-borough.solana-mainnet.discover.quiknode.pro/38cf24edefbebeb60eb7516eff40f076ac0823af/',
```

Comment:
Think we need to purge this here too: https://github.com/Dual-Finance/status/blob/6a4738f878e2992ee38b4961c9e52b334fa1bdd4/app/src/config/config.ts#L10

Reply:
It is a client side fetch. Cant hide it

This change isnt about hiding it, it is about letting us use a parameter for the overnight runs when we run out of quota in a couple days

Reply:
Ok are you worried about ppl scraping for private RPCs elsewhere?

Reply:
Cant avoid it easily, so nothing to really worry about. But should keep it all separate so we have failure isolation.

Reply:
Isolated by using a separate RPC for status/landing/app?

Reply:
Correct. Each type of caller shouldnt be able to cause failures on another due to resource exhaustion when possible